js/reference.js

```diff
@@ -3626,6 +3626,41 @@
             return this._googleDatasetMetadata;
         },
 
+        /**
+         * Check the catalog, schema, and table to see if the saved query UI should show
+         *  1. check the table to see if the value is a boolean
+         *    a. if true or false, we are done and use that value
+         *    b. if undefined, display annotation defined but show_saved_query property was undefined
+         *    c. if null, display annotation was not defined (initial value)
+         *  2. check the schema to see if the value is a boolean
+         *    a. if true or false, we are done and use that value
+         *    b. if undefined, display annotation defined but show_saved_query property was undefined
+         *    c. if null, display annotation was not defined (initial value)
+         *  3. check the catalog to see if the value is a boolean
+         *    a. if true or false, we are done and use that value
+         *    b. if undefined, display annotation defined but show_saved_query property was undefined
+         *    c. if null, display annotation was not defined (initial value)
+         *  4. default to false if not defined on any of the above
+         */
+        get showSavedQuery() {
```

Comment:
Let's move this to be inside the `Reference.display` API as all the other "display" settings are there. So then chaise would have to do `reference.display.showSavedQuery`